docker-compose.yml

```diff
@@ -0,0 +1,133 @@
+# Pull (Pre-Built):
+#   $ podman-compose pull
+# Update & Run:
+#   $ podman-compose pull; \
+#     podman-compose --project-name janitor down; \
+#     podman-compose --project-name janitor up --detach --remove-orphans; \
+#     podman-compose --project-name janitor logs --follow
+# Build:
+#   $ podman-compose build
+# Build & Run:
+#   $ podman-compose --project-name janitor up --build --force-recreate
+# Run:
+#   $ podman-compose --project-name janitor up
+# Debug:
+#   $ podman-compose --project-name janitor logs
+#   $ podman exec -it janitor-archive bash
+#   $ podman exec -it janitor-bzr_store bash
+#   $ podman exec -it janitor-differ bash
+#   $ podman exec -it janitor-git_store bash
+#   $ podman exec -it janitor-ognibuild_dep bash
+#   $ podman exec -it janitor-publish bash
+#   $ podman exec -it janitor-runner bash
+#   $ podman exec -it janitor-site bash
+#   $ podman exec -it janitor-worker bash
+# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+
+services:
```

Comment:
Can you split podman-compose out into a separate PR? I don't think this will work as-is, and it's another thing that can get out of date.

Reply:
Noted - done!